src/components/table/table.stories.js

```diff
@@ -185,7 +176,15 @@ storiesOf('Table', module)
       store.set({ currentPage: setCurrentPage(props) });
       props.size = select('size', OptionsHelper.tableSizes, Table.defaultProps.size);
       props.isZebra = boolean('zebra striping', false);
-
+      props.theme = select(
+        'theme',
+        [
```

Comment:
Is there a reason you can't just pass in the whole `OptionsHelper.tableThemes` object?

Reply:
Yeah haha no idea why I've done that here, the classic story only has two variants but the DLS has them all 🤦‍♂